### DISCUSSION TITLE
Most likely due to a circular import

### DISCUSSION THREAD

**Discussion by kadirnar** [obss/sahi#674] [Q&A] [answered]
Merhaba  @fcakyon Hocam,

https://github.com/obss/sahi/blob/main/sahi/utils/torch.py#L5-L7 torch.py dosyasını çalıştırırken kütüphanelerde hata veriyor.

Error Message:
```
ImportError: cannot import name 'stack' from partially initialized module 'torch' (most likely due to a circular import) 
``` 
İlk önce sürüm ile ilgilii olduğunu düşündüm. Torch'un Lts sürümlerini yüklediğimde aynı hata veriyor. Dosya ismini torch_utils değiştirdiğimde hata vermedi. Kütüphanelerin çakıştığından böyle oluyor galiba.

**Comment by fcakyon** [comment, marked as answer]
@kadirnar torch.py modulunu direkt olarak run etmeyi desteklemiyoruz. Ana proje dizininden import ederek kullanilmasi gerekiyor. Ornek kullanim su sekilde olabilir:

```python
from sahi.utils.torch import cuda_is_available

result = cuda_is_available()
```
